draft-claise-semver.md

```diff
@@ -87,24 +86,23 @@ semver.org):
 * BETA is incremented when a new pre-release or testing version is made. 
 
 
-In IETF terms, this versioning scheme provides functionality equivalent to
+In IETF terms, this versioning scheme provides functionality analagous to
 several parts of the traditional IETF process.
 
-* MAJOR replaces the "obsoletes" relation between RFCs
-* MINOR replaces the "updates" relation between RFCs
```

Comment:
I think I might just water down the wording here and say that these are somehow loosely analogous to these functions.